### ISSUE TITLE
ML example is failing due to strange keras error

### ISSUE THREAD

**Issue by skearnes** [open-reaction-database/ord-schema#616]
https://github.com/open-reaction-database/ord-schema/runs/4097453030?check_suite_focus=true

Reproduces locally as well. Seems totally bizarre.

**Comment by michaelmaser** [comment]
Notebook running fine for me locally. Believe has to do with a lone keras version installed while importing keras in tensorflow at a different version. See [https://stackoverflow.com/questions/58012741/error-importing-tensorflow-alreadyexistserror-another-metric-with-the-same-nam](url)

**Comment by skearnes** [comment]
Blocked by https://github.com/ipython/ipython/issues/13255 (thanks @brilee)

**Comment by skearnes** [comment]
Haven't seen this in a while; marking as resolved.
